examples/times-two/test.smt2

```diff
@@ -0,0 +1,5 @@
+(declare-const x (_ BitVec 32))
```

Comment:
Is everything in the times-two folder about multiplying a variable by 2? Maybe let's just add a README for it to the folder.

Reply:
Done.